diff --git a/node_modules/game7-bridge-sdk/dist/bridgeTransfer.js b/node_modules/game7-bridge-sdk/dist/bridgeTransfer.js
index 9ab8926..24a757a 100644
--- a/node_modules/game7-bridge-sdk/dist/bridgeTransfer.js
+++ b/node_modules/game7-bridge-sdk/dist/bridgeTransfer.js
@@ -272,11 +272,15 @@ export class BridgeTransfer {
     execute(signer) {
         return __awaiter(this, void 0, void 0, function* () {
             if (!this.childTransactionReceipt) {
+                console.log('here before originreceipt')
                 const originReceipt = yield this.originProvider.getTransactionReceipt(this.txHash);
                 this.childTransactionReceipt = new ChildTransactionReceipt(originReceipt);
+                console.log('here after originreceipt')
             }
+            console.log('here')
             const messages = (yield this.childTransactionReceipt.getChildToParentMessages(signer));
             const message = messages[0];
+            console.log(message);
             const res = yield message.execute(this.originProvider);
             return yield res.wait();
         });
